OOStepByStep/Person.cs

```diff
@@ -0,0 +1,121 @@
+﻿namespace OOStepByStep
+{
+    using System;
+    using System.Collections.Generic;
+
+    public class SchoolClass
+    {
+        private string name;
+        private Teacher teacher = null;
```

Comment:
this is not necessary, the default value for object is null

Reply:
But it is good habit to always initialize your variable.